paddle/fluid/eager/auto_code_generator/generator/eager_gen.py

```diff
@@ -1839,6 +1839,13 @@ def GenerateHigherOrderNodeCreationCode(self):
             False if self.composite_func_info == {} else True
         )
 
+        if is_composite_grad_api and next_grad_node_creation_str != '':
```

Comment:
what if is_composite_grad_api and next_grad_node_creation_str is ''

Reply:
it will be delete when delete prim_enable flag